tests/lib.test.js

```diff
@@ -1,83 +1,89 @@
-'use strict';
-
-var assert = require('assert')
+const test = require('ava')
 
 var config = require('../config')
 var lib = require('../lib/')
 var mocks = require('./fixtures')
 
 mocks.initMocks()
 
-describe('Lib', () => {
+test('Gets datapackage.json', async t => {
   const api = new lib.DataHubApi(config)
```

Comment:
Can you move all `const api = new lib.DataHubApi(config)` outside somewhere in line 8, so we don't need to instantiate everytime.